.travis.yml

```diff
@@ -1,7 +1,7 @@
 # System information
 os: linux
 addons:
-  postgresql: "9.3"
+  postgresql: "12.2"
```

Comment:
Does psycopg2-binary needs this specific postgresql version or this is just for using the latest ?

Reply:
psycopg2 works for Postgres 7.4 to Postgres 12.

Also, why pscopg2-binary and not psycopg2? Ans: https://www.postgresql.org/message-id/CA%2Bmi_8bd6kJHLTGkuyHSnqcgDrJ1uHgQWvXCKQFD3tPQBUa2Bw%40mail.gmail.com